openverse_catalog/dags/common/loader/reporting.py

```diff
@@ -104,10 +149,19 @@ def report_completion(
 
     # Collect data into a single message
     message = f"""
-*Provider*: `{provider_name}`
+*DAG*: `{dag_id}`
```

Comment:
Previously we were already using dag_id, and just chopping `_workflow` off (ie `dag_id.replace("_workflow", "")`). We could continue doing so, but then for reingestion flows we'd get `wikimedia_reingestion` as the "provider", which isn't quite correct.

I can think of other things to do but they all seem like overkill to me. We could look for `reingestion` in the dag_id, but we'd still need to get the name of the actual _provider_. That would mean making a mapping somewhere, or else passing the information all the way through from the ingester class. I guess we could chop off `workflow` _if 'reingestion' isn't in the dagid_, and then report "provider" for regular flows and "dag_id" for reingestion 🤔

Reply:
Perhaps on the `ProviderWorkflow` dataclass we could specify `provider` here instead of `dag_id`?
https://github.com/WordPress/openverse-catalog/blob/18decf9b70f409bc83809d14b7be3b65cb91d409/openverse_catalog/dags/providers/provider_workflows.py#L73

Then we could also pass in the workflow type (regular/reingestion) to the reporting function which could be used to craft the message. On the other hand though, that all sounds like a lot of work so maybe DAG ID is just fine 😅